src/xcode/ENA/ENA/Source/AppDelegate & Globals/__tests__/RouteTests.swift

```diff
@@ -23,7 +23,47 @@ class RouteTests: CWATestCase {
 		case .success:
 			break
 		case .failure:
-			XCTFail("Error was nor intended")
+			XCTFail("Error was not intended")
+		}
+	}
+
+	func testGIVEN_validRATUrlWithDGCTrue_WHEN_parseRoute_THEN_isValid() {
+		// GIVEN
+		let validRATTestURL = "https://s.coronawarn.app?v=1#eyJ0aW1lc3RhbXAiOjE2MjI2MzA0NzAsInNhbHQiOiIzNDJGRDRGM0RFNzQwNjI5RjlDOTdGMkJCODUxMUJBQyIsInRlc3RpZCI6ImIxYmNkZWYzLTNjZGEtNDI0NS05ZTk2LTZkNzkzZWE4YjYwZCIsImhhc2giOiI2ODJlMzNmZDc3YmY1NzE5ZWYxNzZmYjRlN2ZjNzIzNzQ4NmQ2OGZjMTVkMDE2OTJjMDA0OWEyNGM2ZmE2NTYzIiwiZm4iOiJDbHlkZSIsImxuIjoiTW9udGlnaWFuaSIsImRvYiI6IjE5NTktMTAtMDIiLCJkZ2MiOnRydWV9"
+
+		// WHEN
+		let route = Route(validRATTestURL)
+		guard case let .rapidAntigen(result) = route else {
+			XCTFail("unexpected route type")
+			return
+		}
+
+		// THEN
+		switch result {
+		case .success:
+			break
```

Comment:
Maybe add a check if this is a route with a point of care value?

Reply:
Added more tests for the content of the parsed route now, including the new property